packages/supermassive/src/executeWithSchema.ts

```diff
@@ -20,7 +19,7 @@ export function executeWithSchema({
   fieldResolver,
   typeResolver,
 }: ExecutionWithSchemaArgs): PromiseOrValue<ExecutionResult> {
-  const schema = makeExecutableSchema({ typeDefs, resolvers });
+  const schema = buildASTSchema(typeDefs);
```

Comment:
I'm unsure why this is included here, was this intentional?

Reply:
Another react like example that was breaking for office start.